packages/react-draggable-dialog/src/components/DraggableDialog/DraggableDialog.component-browser-spec.tsx

```diff
@@ -1,13 +1,10 @@
 import * as React from 'react';
 import { test, expect } from '@playwright/experimental-ct-react';
 
-const Example = () => {
-  return <div>Example</div>;
-};
-
 test.use({ viewport: { width: 500, height: 500 } });
 
 test('should render basic component', async ({ mount }) => {
-  const component = await mount(<Example />);
+  const component = await mount(<div>Example</div>);
```

Comment:
looks like that previous pattern used wasn't capturing this test, thus we didn't run into any errors. now we do

https://github.com/microsoft/playwright/issues/18057